lesson4/decision.md

```diff
@@ -52,13 +52,13 @@ if (<CONDITION>) {
 
 ### The Condition
 
-The condition of an `if`-statement has to be inside round parentheses. The CONDITION part has to be a boolean expression,
+The condition of an `if` statement has to be inside round parentheses. The CONDITION part has to be a boolean expression,
```

Comment:
I would drop `round` here. Maybe just an example after parentheses in case it's a new term? "The condition of an `if` statement has to be inside parentheses, e.g. `if (1 < 2)`"